docs/blog/community-showcase-q2.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Community Showcase (Q2 2025)
+author:
+  - name: The e18e Contributors
+sidebar: false
+date: 2025-07-17
+head:
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: og:title
+      content: Community Showcase (Q2 2025)
+  - - meta
+    - property: og:url
+      content: https://e18e.dev/blog/community-showcase-q2
+  - - meta
+    - property: og:description
+      content: An update on what the community has been up to in Q2 of 2025
+  - - meta
+    - property: og:image
+      content: https://e18e.dev/og/community-showcase-25-q1.png
+---
+
+_July 17, 2025_
+
+# ![Community Showcase (Q2 2025)](/og/community-showcase-25-q1.png)
+
+A little late, but it's time for another community showcase! :tada:
+
+So many good things have happened over the last quarter in the community and around the ecosystem, and there's so much more to come :rocket:
+
+## Prettier CLI
+
+The Prettier team has been collaborating with the e18e community for some time now on a new, faster CLI for Prettier. In June, this was released behind a flag for all to try out! :tada:
+
+This has been a great collaboration, driven mostly by the work of [@fabiospampinato](https://bsky.app/profile/fabiospampinato.bsky.social) and plenty of help from [@fisker](https://bsky.app/profile/fiskercheung.com) on the Prettier side of things.
+
+You can see some of the huge performance improvements in the chart below:
+
+![prettier run time results](/images/prettier-chart.png)
+
+You can use this in your own project by using the new experimental flag:
+
+```sh
+# via the CLI flag
+prettier . --check --experimental-cli
+
+# via an environment variable
+PRETTIER_EXPERIMENTAL_CLI=1 prettier . --check
+```
+
+Without any other changes, this should speed up your formatting runs a whole lot :raised_hands:
+
+Read more about how we got here in the [blog post](https://e18e.dev/blog/prettier-speed-up.html).
+
+## e18e CLI
+
+We posted about the upcoming e18e CLI some time ago and the rough idea behind it. Since then, the community has been hard at work building it and shaping the definition of what's to come.
+
+Soon you will be able to do cool things like this:
+
+```sh
+npx @e18e/cli migrate chalk # automatically migrate chalk to picocolors
+
+npx @e18e/cli migrate --interactive chalk # interactively choose which library to migrate chalk to
+
+npx @e18e/cli analyze # lint your dependency tree and produce useful stats
+```
+
+We now have the basics in place and a much fuller list of features to work towards. Just a few of the things this CLI will be able to do:
+
+- Automatic migrations to suggested replacements
+- Lint your dependency tree for common issues
+- Lint your publishing setup (with help from Publint!)
```

Comment:
```suggestion
- Lint your publishing setup (with help from publint!)
```
nit: The maintainers seem to prefer the lowercase variant based on their README.